deal.II-toolchain/packages/petsc.package

```diff
@@ -104,7 +121,7 @@ package_specific_setup () {
 
 package_specific_register () {
     export PETSC_DIR=${INSTALL_PATH}
-    if [ ! -z "${PARMETIS_DIR}" ]; then
+    if [ ! -z "${PARMETIS_DIR}" ] && [ -z "${SCALAPACK_DIR}" ]; then
         export SCALAPACK_DIR=${INSTALL_PATH}
```

Comment:
there should be a comment that scalapack is installed through parmetis

Reply:
sorry, petsc should be the one installing scalapack

Reply:
Why should be ScaLAPACK installed via PETSc?

Reply:
Your statement is: "if parmetis and not scalapack", therefore petsc installs scalapack (and `INSTALL_PATH` points to petsc here.

Reply:
If ScaLAPACK is already installed and therefore `SCALAPACK_DIR` is already specified, this check should ensures that `SCALAPACK_DIR` is not overwritten.
So I don't see my mistake at the moment....

Reply:
you didn't inverted the statement for scalapack, so you should use "!" in front, or, use "- n" instead.

Reply:
I added the missing `!` in `[ -z "${SCALAPACK_DIR}" ]`